common/flatpak-dir.c

```diff
@@ -7584,6 +7584,15 @@ export_desktop_file (const char         *app,
       /* Add a marker so consumers can easily find out that this launches a sandbox */
       g_key_file_set_string (keyfile, G_KEY_FILE_DESKTOP_GROUP, "X-Flatpak", app);
 
+      /* Disable krunner dbusplugins by default, so that flatpak applications cannot
+       * unintentionally grab sensible search data.
```

Comment:
```suggestion
       * unintentionally grab sensitive search data.
```

(assuming you mean "sensitive" = user might prefer to keep this secret, rather than "sensible" = reasonable/normal/rational)